exchanges/gateio/gateio_test.go

```diff
@@ -1835,7 +1835,22 @@ func TestGetSettlementHistory(t *testing.T) {
 
 func TestGetOptionsSpecifiedSettlementHistory(t *testing.T) {
 	t.Parallel()
-	if _, err := g.GetOptionsSpecifiedSettlementHistory(context.Background(), "BTC_USDT-20221028-40000-C", "BTC_USDT", 0); err != nil && !strings.Contains(err.Error(), "NOT_FOUND") {
+	pairs, err := g.FetchTradablePairs(context.Background(), asset.Options)
+	if err != nil {
+		t.Skip(err)
+	}
+	if len(pairs) == 0 {
+		t.Skip(err)
+	}
+	pair, err := currency.NewPairFromString(pairs[1])
+	if err != nil {
+		t.Skip(err)
+	}
+	underlying, err := g.GetUnderlyingFromCurrencyPair(pair)
+	if err != nil {
+		t.Skip(err)
+	}
+	if _, err := g.GetOptionsSpecifiedSettlementHistory(context.Background(), pairs[1], underlying, 0); err != nil {
```

Comment:
This is still not working. From request [HERE](https://github.com/thrasher-corp/gocryptotrader/pull/1058#discussion_r1022280321)

This endpoint does not require a `contract string` param, please remove that completely with the `/` it only required the underlying 

```
GateIO attempt 1 request path: https://api.gateio.ws/api/v4/options/settlements/BTC_USDT-20221125-26000-C?at=0&underlying=BTC_USDT
GateIO request type: GET

GateIO GetOptionsSpecifiedSettlementHistory() error GateIO unsuccessful HTTP status code: 404 raw response: {"label":"NOT_FOUND"}
```